src/stylesheets/elements/_tags.scss

```diff
@@ -1,26 +1,24 @@
 .sds-tag--info-purple {
```

Comment:
colors should be added by using the `color()` function example `color: color('primary');`

Reply:
Updated to mixins